cmd/terramate/cli/cloud.go

```diff
@@ -683,7 +679,7 @@ func (c *cli) detectGithubMetadata(owner, reponame string) {
 	ghToken, tokenSource := auth.TokenForHost(githubDomain)
 	if ghToken == "" {
 		printer.Stderr.WarnWithDetails(
-			"Export GITHUB_TOKEN with your GitHub credentials for enabling metadata collection",
+			"Exporty GITHUB_TOKEN with your GitHub credentials for enabling metadata collection",
```

Comment:
```suggestion
			"Export GITHUB_TOKEN with your GitHub credentials for enabling metadata collection",
```

Reply:
this was a "test push" to trigger a problem, I should have added a comment, sorry.
Thanks for spotting it anyway :)